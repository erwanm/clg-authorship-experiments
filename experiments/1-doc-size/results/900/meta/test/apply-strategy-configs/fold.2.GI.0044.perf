0.573488	0.785600	0.730000
