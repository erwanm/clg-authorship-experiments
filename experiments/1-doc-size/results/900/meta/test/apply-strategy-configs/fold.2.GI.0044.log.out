apply-config.sh: applying config '/tmp/expe1/900/process/selected-meta-configs/fold.2.0095.model/strategy-configs/fold.2.GI.0044' to cases '/home/moreaue/dump/expe1//results/900/meta/test/apply-strategy-configs/only-cases.tmp.BpEIkQKv'; results written to '/home/moreaue/dump/expe1//results/900/meta/test/apply-strategy-configs/fold.2.GI.0044'
apply-config.sh: done.
